app/helpers/fact_values_helper.rb

```diff
@@ -21,13 +22,13 @@ def fact_name(value, parent)
             url = fact_values_path(parameters)
           end
           link_to(current_name, url,
-                  :title => _("Show all %s children fact values") % value_name)
+                  :title => _("Show all %s children fact values") % memo)
```

Comment:
Note this actually fixes an issue where it was always showing "Show all grandparent.parent.child fact values"   rather than "Show all grandparent fact values" and "Show all grandparent.parent" fact values